src/main/java/edu/ucsb/cs156/spring/backenddemo/services/CountryCodeQueryService.java

```diff
@@ -40,7 +40,7 @@ public String getJSON(String country) throws HttpClientErrorException {
 
         Map<String, String> uriVariables = Map.of("country", country);
 
-        HttpEntity<String> entity = new HttpEntity<>("body", headers);
+        HttpEntity<String> entity = new HttpEntity<>(headers);
```

Comment:
I don't think we should modify the CountryCodeQueryService file unless it fails some test

Reply:
Woops, yeah I think I edited this on accident